src/scripts.js

```diff
@@ -124,11 +124,24 @@ changeBackground.addEventListener("click", () => {
 });
 
 promiseAll().then((responses) => {
+  assignData(responses);
+  createUser();
+  createClasses(user);
```

Comment:
Here you are passing in user as argument but in line 140 createUser function you are not, is this a typo or on purpose?

Reply:
done!